src/Catalyst.Core.Lib/P2P/IO/Observers/DeltaHeightRequestObserver.cs

```diff
@@ -73,10 +73,12 @@ public sealed class DeltaHeightRequestObserver
 
             var deltaIndexDao = _deltaIndexService.LatestDeltaIndex();
             var deltaIndex = deltaIndexDao.ToProtoBuff<DeltaIndexDao, DeltaIndex>(_mapperProvider);
-            return new LatestDeltaHashResponse
-            {
-                Result = deltaIndex
-            };
+            return new LatestDeltaHashResponse();
+
```

Comment:
something changed here and this is an issue to resolve

Reply:
@Xela101 this might be related to your sync work